src/citrine/resources/predictor_evaluation_execution.py

```diff
@@ -114,7 +114,7 @@ class PredictorEvaluationExecutionCollection(Collection["PredictorEvaluationExec
     _collection_key = 'response'
     _resource = PredictorEvaluationExecution
 
-    def __init__(self, *,
+    def __init__(self,
```

Comment:
I was in general not sure if we want kwargs for Collections, since those aren't created by users generally

Reply:
I think not, since they're internal